lib/Elements/Element.tsx

```
@@ -0,0 +1,373 @@
+                <Text style={[streamdown.theme.td.base, { textAlign: "left" }]}>
+                  {children}
+                </Text>
+              </View>
+            </Slot>
+          );
+        }
+        return null;
+
+      case "th":
+        const thToken = token as any;
+        if (thToken.rowspan > 0) {
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)